aexpect/client.py

```diff
@@ -54,6 +54,8 @@
 
 _THREAD_KILL_REQUESTED = threading.Event()
 
+log = logging.getLogger(__name__)
```

Comment:
My only suggestion here is to rename this constant to LOG instead of log to follow pep8.

Reply:
Thanks @beraldoleal for the review. Can you point me to the PEP8 guideline about this? I think the pushed branched passed multiple checks, including PEP8 ones. Also, we discussed the same option with @willianrampazzo in https://github.com/avocado-framework/aexpect/pull/89#discussion_r747521840 and settled on the fact that this is optional. Finally, I already edited all all messages so I am somewhat unwilling to go through the entire thing again but I understand that this might be preferred for `log` to be deemed a constant, just wondering how solid this potential PEP8 guideline is.

Reply:
https://www.python.org/dev/peps/pep-0008/#constants

It is just a suggestion, not a blocker.

Reply:
No worries, I will go through it one more time with some IDE replacements. I rather meant redoing this as problematic because on my side it also involves backpropagating patches to other fork branches as well as RPMs. As there was no further input from last time with Willian I thought the only justification for using `LOG` is compatibility with Avocado projects and decided to go with a different final change anyways. The PEP8 provides further justification that I didn't have at the time.